.azure-pipelines/templates/windows-functional-test.yml

```diff
@@ -0,0 +1,34 @@
+steps:
+
+  - task: DownloadBuildArtifacts@0
+    displayName: Download functional test enlistment
+    inputs:
+      buildType: current
+      downloadType: specific
+      artifactName: FunctionalTests_$(platformFriendlyName)_$(configuration)

```

Comment:
I think you mentioned this and looking at this, it looks like it is tied to the current build's artifacts so we don't need the build number in the `artifactName` right?

Reply:
Thinking about this a little bit though and what if I want to download artifacts to my machine?  I guess there will only be one and I will need to remember, rename, or check the date to know what build it was?  Just wondering if it would hurt anything to put the build number in the artifact name.

Reply:
@kewillford I decided to not include the build number because we don't do that for other artifacts like SetupGVFS.exe. Since you already need a build number to get an artifact, I opted to avoid the duplication of that information.